connect.php

```diff
@@ -4,7 +4,7 @@
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
-$pass = DBPASSWORD;
+$pass = '12345678';
```

Comment:
1. There has been a slight modification on how to setup the database. 
Follow the instruction mentioned on [contribution.md](https://github.com/Infinito-IIT-Patna/Infinito2020/blob/master/CONTRIBUTING.md).
2. The image file is about 7mb which is pretty large. Compress the image and add it again.